pkg/storage/legacysql/dualwrite/dualwriter_mode1_test.go

```diff
@@ -90,63 +94,6 @@ func TestMode1_Create(t *testing.T) {
 	}
 }
 
-func TestMode1_CreateOnUnifiedStorage(t *testing.T) {
```

Comment:
This test removes all the parts that are explicitly testing the previous utility function that would check if unified is actually called.

Since we are now using the *same* struct for all three paths, I think we have good enough coverage to know that we can call both legacy+unified